f1trklstarray.php

```diff
@@ -0,0 +1,23 @@
+<?php
+$trcklst = [
```

Comment:
Rather then expose this array as global I'd suggest to wrap it in a function which can be used everywhere:
```
function getTrackList():array
{
    return [...];
}
```